src/main/java/org/apache/sling/scripting/core/impl/jsr223/SlingScriptEngineManager.java

```diff
@@ -72,8 +76,21 @@
         policy = ReferencePolicy.DYNAMIC
     )
 )
+@Designate(ocd=SlingScriptEngineManager.Config.class)
 public class SlingScriptEngineManager extends ScriptEngineManager implements BundleListener {
 
+    @ObjectClassDefinition(name ="Apache Sling Script Engine Manager",
+            description="Configures options for the Script Engine Manager")
+    public @interface Config {
+
+        @AttributeDefinition(name = "Includes", description = "Regular expressions to match script engines to include (matched against short names).")
```

Comment:
I'd say "A script engine with a short name that matches any of these expressions is activated"

Reply:
Would you be ok with "is included" instead of "is activated"?  I don't want to imply that this configuration is having any impact on whether a ScriptEngine in active in the greater JVM.  The setting only affects whether those are utilized in this particular context.

Reply:
Sure, "is included" works! And same for the ignores of course. This is just a suggestion anyway but it felt clearer to me.